src/frontend/src/lib/components/send/SendTokensList.svelte

```diff
@@ -26,6 +26,10 @@
 	{/each}
 </TokensSkeletons>
 
+{#if tokens.length === 0}
+	<p class="mt-4 text-dark opacity-50">{$i18n.tokens.manage.text.all_tokens_zero_balance}</p>
```

Comment:
Can you provide a screenshot? Is that visible? Do we have other text that are displayed with an `opacity-50` or that's an on-purpose choice?

Reply:
Sure, I included it in the comment (and I included a bit of margin at the bottom).

As per the style I am using the same as the `TokensSignedIn` list: the warning appear when the user hides zero balances and all the tokens have null balance.

Reply:
Gotcha! Thanks